data-collections-exercises.md

```diff
@@ -3,95 +3,96 @@
 Starting with the following array...
 
 ```rb
-planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
+#planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 ```
 
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+#  planeteers[1]
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+#  planeteers.collect {|p| p + " Heart" }
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+# planeteers.delete("Captain Planet") 
```

Comment:
you used a method but you solved the problem you can use `-` operator to delete an element without using a method `planeteers - ["Captain Planet"]`

Reply:
oh!
sorry I mixed up between function and method in hash and react